charts/stable/bitwardenrs/templates/configmap.yaml

```
@@ -29,6 +29,9 @@ data:
   {{- if and (not .existingSecret.enabled) .user }}
   SMTP_USERNAME: {{ .user | quote }}
   {{- end }}
+  {{- if .password }}
+  SMTP_PASSWORD: {{ .password | quote }}
+  {{- end }}
```

```suggestion
  {{- if and (not .existingSecret.enabled) .password }}
  SMTP_PASSWORD: {{ .password | quote }}
  {{- end }}
```

Shouldn't it be this?

---

Yes it should be. :+1: